doc/reference/peripherals/peci.rst

```diff
@@ -0,0 +1,28 @@
+.. _peci:
+
+PECI
+####
+
+Overview
+********
+The Platform Environment Control Interface, abbreviated as PECI,
+is a thermal management standard introduced in 2006
+with the Intel Core 2 Duo microprocessors.
+The PECI interface for external devices to read processor temperature,
+perform processor manageability functions, and manage processor interface
+tuning and diagnostics. For support on boards and/or systems
```

Comment:
The PECI bus driver APIs enable the interaction between Embedded Microcontrollers and CPUs.